docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1072,212 +1072,602 @@ thor:
-          Nodes. Using placements you can configure the Kubernetes scheduler
-          to use a "pods" list to apply settings to pods. For example:</para>
+      <para>Define the <emphasis>indexBuildPlane</emphasis> value as a helm
+      chart option to allow index files to be written by default to a
+      different data plane. Unlike flat files, index files have different
+      requirements. The index files benefit from quick random access storage.
+      Ordinarily flat files and index files are output to the defined default
+      data plane(s). Using this option you can define that index files are
+      built on a separate data plane from other common files. This chart value
+      can be supplied at a component or global level.</para>
 
-          <programlisting> placements:
+      <para>For example, adding the value to a global level under
+      globlal.storage :</para>
+
+      <programlisting>global:
+  storage:
+    indexBuildPlane: myindexplane</programlisting>
+
+      <para>Optionally, you could add it at the component level, as
+      follows:</para>
+
+      <programlisting>thor:
+- name: thor
+  prefix: thor
+  numWorkers: 2
+  maxJobs: 4
+  maxGraphs: 2
+  indexBuildPlane: myindexplane</programlisting>
+
+      <para>When this value is set at the component level it would override
+      the value set at the global level.</para>
+    </sect2>
+  </sect1>
+
+  <sect1 id="CV_Pods-Nodes">
+    <title>Pods and Nodes</title>
+
+    <para>One of the key features of Kubernetes is its ability to schedule
+    pods on to nodes in the cluster. A pod is the smallest and simplest unit
+    in the Kubernetes environment that you can create or deploy. A node is
+    either a physical or virtual "worker" machine in Kubernetes.</para>
+
+    <para>The task of scheduling pods to specific nodes in the cluster is
+    handled by the kube-scheduler. The default behavior of this component is
+    to filter nodes based on the resource requests and limits of each
+    container in the created pod. Feasible nodes are then scored to find the
+    best candidate for the pod placement. The scheduler also takes into
+    account other factors such as pod affinity and anti-affinity, taints and
+    tolerations, pod topology spread constraints, and the node selector
+    labels. The scheduler can be configured to use these different algorithms
+    and policies to optimize the pod placement according to your cluster’s
+    needs.</para>
+
+    <para>You can deploy these values either using the values.yaml file or you
+    can place into an file and have Kubernetes instead read the values from
+    the supplied file. See the above section <emphasis>Customization
+    Techniques</emphasis> for more information about customizing your
+    deployment.</para>
+
+    <sect2 id="YAML_FileStruct_Placement">
+      <title>Placements</title>
+
+      <para>The Placements is a term used by HPCC Systems, which Kubernetes
```

Comment:
The Placements term is not our term, it is a standard to Kubernetes. Please reword